_site/index.html

```diff
@@ -0,0 +1,214 @@
+
+<h3 id="how-does-this-relate-to-semver">How does this relate to SemVer?</h3>
+
+<p><code class="highlighter-rouge">fix</code> type commits should be translated to <code class="highlighter-rouge">PATCH</code> releases. <code class="highlighter-rouge">feat</code> type commits should be translated to <code class="highlighter-rouge">MINOR</code> releases. Commits with <code class="highlighter-rouge">BREAKING CHANGE</code> in the commits, regardless of type, should be translated to <code class="highlighter-rouge">MAJOR</code> releases.</p>
+
+<h3 id="how-should-i-version-my-commit-type-specification-extensions-eg-jameswomackconventional-commit-spec">How should I version my commit type specification extensions, e.g. <code class="highlighter-rouge">@jameswomack/conventional-commit-spec</code>?</h3>
+
+<p>We recommend using SemVer and publishing to an NPM-compatible registry.</p>
+
+<h3 id="what-do-i-do-if-i-accidentally-use-the-wrong-commit-type">What do I do if I accidentally use the wrong commit type?</h3>
+
+<h4 id="when-you-used-a-type-thats-of-the-spec-but-not-the-correct-type-eg-fix-instead-of-feat">When you used a type that’s of the spec but not the correct type, e.g. <code class="highlighter-rouge">fix</code> instead of <code class="highlighter-rouge">feat</code></h4>
+
+<p>Prior to merging or releasing the mistake, we recommend using <code class="highlighter-rouge">git rebase -i</code> to edit the commit history. After release, the cleanup will be different according to what tools and processes you use.</p>
+
+<h4 id="when-you-used-a-type-not-of-the-spec-eg-feet-instead-of-feat">When you used a type <em>not</em> of the spec, e.g. <code class="highlighter-rouge">feet</code> instead of <code class="highlighter-rouge">feat</code></h4>
+
+<p>In a worst case scenario, it’s not the end of the world if a commit lands that does not meet the conventional commit specification. It simply means that commit will be missed by tools that are based on the standard.</p>
+
+<h3 id="do-all-my-contributors-need-to-use-the-conventional-commit-specification">Do all my contributors need to use the conventional commit specification?</h3>
+
+<p>No! If you use a squash based workflow on Git lead maintainers can cleanup the commit messages as they’re merged—adding no workload to casual committers. A common workflow for this is to have your git system automatically squash commits from a pull request and present a form for the lead maintainer to enter the proper git commit message for the merge.</p>
+
+<h2 id="whos-behind-thisinspired-this">Who’s Behind This/Inspired This?</h2>
+
+<p>The Conventional Commit standard is inspired by the <a href="https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit">Angular Commit Guidelines</a>, which
```

Comment:
👍 feel free to submit a patch with this; otherwise I'll do it after the work day today.